content/en/docs/polo-features/exports/index.md

```diff
@@ -33,11 +29,15 @@ Finally there is a Full ADIF Export which does contain all the data form the ope
 PoLo supports these measures and recommends that you use each export log for their intended purpose.
 
 ### Overrides
-
 If you must, there is an override to this default behaviour.
 
 ![image](./exportprivatedata.png)
 
 In General Settings|Export Settings, it is possible to enable the "Include Private Data" option. This option will cause the chosen export file to include the personal data of each QSO operator.
 
-Please use this option with caution. Peronal lookup data is, of course, subject to the lookup options you have enabled and subscribed to elsewhere in PoLo and also to the data you manually enter into the "More" tab of the QSOs in question. Remember that once you pass on someone's personal data, you and they no longer have control over who else can obtain it. Habits which protect other people's data will also therefore, protect your own.
\ No newline at end of file
+Please use this option with caution. Peronal lookup data is, of course, subject to the lookup options you have enabled and subscribed to elsewhere in PoLo and also to the data you manually enter into the "More" tab of the QSOs in question. Remember that once you pass on someone's personal data, you and they no longer have control over who else can obtain it. Habits which protect other people's data will also therefore, protect your own.
+
+### General Settings - Privacy
```

Comment:
lets remove this from this page.

Maybe include it in the general "Logger Settings" page?